### PR TITLE
Pair programming

### PR DESCRIPTION
**Pull request by wuletawwonte**
Changes in this Milestone
- Added contact form section
- Added HTML5 form validation
- Submit the form to formspree service

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -162,6 +164,33 @@ <h2>Skills</h2>
...
+              <label>Your name</label>
+              <input required type="text" maxlength="30" name="name" placeholder="Your name">
+            </div>
+
+            <div class="email-container labeled-element">
+              <label>Your email</label>
+              <input required type="email" name="email" placeholder="Your email">
+            </div>
+
+            <textarea required class="labeled-element" name="message" placeholder="Enter your message here..."></textarea>
```
Comment (micheaol) [inline index.html]: Please ensure that the message input has maxlength="500" attribute according to the requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -162,6 +164,33 @@ <h2>Skills</h2>
...
+              <label>Your name</label>
+              <input required type="text" maxlength="30" name="name" placeholder="Your name">
+            </div>
+
+            <div class="email-container labeled-element">
+              <label>Your email</label>
+              <input required type="email" name="email" placeholder="Your email">
+            </div>
+
+            <textarea required class="labeled-element" name="message" maxlength="500" placeholder="Enter your message here..."></textarea>
```
Comment (mateo951) [inline index.html:186]: Addressed 😎

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done making the changes requested by the previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**